src/tokens-studio/spectrum-non-colors/spectrum/typography.component/mobile.json

```diff
@@ -1239,6 +1239,46 @@
       }
     }
   },
+  "detail-cjk-size-xl": {
+    "value": "{font-size-100}",
+    "type": "fontSizes",
+    "$extensions": {
+      "spectrum-tokens": {
+        "uuid": "",
+        "name": "detail-cjk-size-xl"
+      }
+    }
+  },
+  "detail-cjk-size-l": {
+    "value": "{font-size-75}",
+    "type": "fontSizes",
+    "$extensions": {
+      "spectrum-tokens": {
+        "uuid": "",
+        "name": "detail-cjk-size-l"
+      }
+    }
+  },
+  "detail-cjk-size-m": {
+    "value": "{font-size-50}",
+    "type": "fontSizes",
+    "$extensions": {
+      "spectrum-tokens": {
+        "uuid": "",
+        "name": "detail-cjk-size-m"
+      }
+    }
+  },
+  "detail-cjk-size-s": {
+    "value": "{font-size-50}",
```

Comment:
I think this one should be `font-size-25`

Reply:
Thanks Lynn, looks like Tokens Studio might automatically apply the next available token if we explicitly add a non-existent token as value in JSON 😅

Reply:
Fixed 👍